### ISSUE TITLE
Categorical exposure

### ISSUE THREAD

**Issue by mhmism** [jchen1981/MedTest#2]
First of all, thank you so much for this amazing article and r package. 

I would like to apply it to my own dataset, but I have a question in mind. 
I see in the used example that exposure (fire intake) is a continuous variable. 

In cases where the exposure is a case/control (categorical) variable, can I transform it into a numerical variable (1 vs 2) and then use the MedTest package. Am I violating assumptions when doing so?

Looking forward for your response!

Kind regards
Mahmoud

**Comment by jchen1981** [comment]
Thanks for your interest. Yes, if it’s binary exposure, you can code it as 0 and 1, and MedTest can still be applied. If it’s multi-category, currently it does not support.

Best,
Jun

**Comment by mhmism** [comment]
Thank you for your response. 
In case it is a ranked variable where the coding does make sense e.g. increasing severity (e.g 0, 1,2). Would it still be applicable?

**Comment by jchen1981** [comment]
Yes. But of course, there is implicit assumption of the linear effect for the exposure, i.e. the difference between level 0 and 1 is the same as that between level 1 and 2.

Jun

**Comment by mhmism** [comment]
Thanks a lot for this clear answer, which is really helpful.

I have one last question and I hope that you can kindly provide me an answer.

in your article "A distance-based approach for testing the mediation effect of the human microbiome", you mentioned that you investigated which individual taxa could play a mediation role. How did you approach this?  

In order to use the MedTest package you need to convert the abundance values for each taxa into a distance? is this correct? in that case which distance did you use? because from my understanding to calculate to certain microbiome distances such as the bray-curtis  you need to information from multiple taxa provided. 

Looking forward to get your feedback!

Kind regards

**Comment by jchen1981** [comment]
For a specific taxon, you can pull out all the OTUs belonging to that taxon and construct some distances. Or you can create a distance based on the abundance of a single taxon. Either way is possible. The first approach is powerful when the OTUs belonging to the same taxon have very heterogenous effects while the latter approach is more powerful to detect more consistent changes across OTUs.

Jun

**Comment by mhmism** [comment]
Thanks a lot. That's helpful.

Kind regards
Mahmoud

**Comment by yolo661798** [comment]
I have a question about the data given. How can I know which taxon the microorganism is in when I know the ID?
您好，我想请教一下有关提供的微生物数据的问题，数据中给出了微生物的ID，我该怎么通过ID知道该微生物对应的所在分类类别呢？谢谢！
